src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+  const message = document.createElement('div');
+
+  message.className = 'message';
+
+  const clickHandle = () => {
+    const text = 'Promise was resolved!';
+
+    message.append(text);
+
+    resolve(message);
+
+    logo.removeEventListener('click', clickHandle);
+  };
+
+  logo.addEventListener('click', clickHandle);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  const message = document.createElement('div');
+
+  message.className = 'message';
+  message.classList.add('error-message');
+
+  const text = 'Promise was rejected!';
+
+  message.append(text);
+
+  setTimeout(() => reject(message), 3000);
+});
+
+promise1.then((result) => {
+  document.body.append(result);
```

Comment:
Appending directly to the body without checking if `result` is a valid DOM element might cause issues. Ensure `result` is a valid element before appending.